test/integration/base_integration_test.h

```diff
@@ -186,15 +186,19 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
       const std::vector<std::string>& expected_resource_names_removed, bool expect_node = false,
       const Protobuf::int32 expected_error_code = Grpc::Status::WellKnownGrpcStatus::Ok,
       const std::string& expected_error_message = "");
+
   template <class T>
   void sendDiscoveryResponse(const std::string& type_url, const std::vector<T>& state_of_the_world,
                              const std::vector<T>& added_or_updated,
-                             const std::vector<std::string>& removed, const std::string& version) {
+                             const std::vector<std::string>& removed, const std::string& version,
+                             bool use_wrapper = false,
+                             const Protobuf::Map<std::string, ProtobufWkt::Any>& metadata = {}) {
```

Comment:
Why is the type `Protobuf::Map`?
A regular flat_hash_map may be more useful here, as it allows inline initialization.
(i.e., the call will be `sendDiscoveryResponse(..., {{"key", ProtobufWkt::Any()}})` or something of that sort

Reply:
Done.